README.asciidoc

```diff
@@ -369,6 +370,29 @@ command names:
 ------------------------------------------------------------------------------------------
 
 
+[[Piping]]
+Hints on using image-bootstrap within a pipe
+--------------------------------------------
+If you want to run *image-bootstrap* in a pipe to capture its output to both
+`stdout/stderr` in a single log file, be sure to run *image-bootstrap* in
+unbuffered mode, e.g.:
+
+----------------------------------------------------
+python -u image-bootstrap [params] 2>&1 | tee my.log
+----------------------------------------------------
+
+The default shebang generated by python setuptools does not use `-u`. It's also
+not easily possible to have it pass `-u` since the `env` command does allow for
+for passing command parameters only in fairly recent versions. As a consequence,
+`stdout/stderr` won't be synchronized and error output you see in a log file
+will not exactly correspond to preceding/succeeding output on `stdout`. (See
+https://github.com/hartwork/image-bootstrap/issues/71 for more details)
```

Comment:
Let's not show the full URI but "issue #71", instead. I believe the syntax is `https://github.com/hartwork/image-bootstrap/issues/71[issue #71]`.
After "details", we could add a full stop.

Reply:
IMO, nope. Someone might be reading this offline, then being cut off...

Reply:
`https://github.com/hartwork/image-bootstrap/issues/71[issue #71]` is the asciidoc equvalent of markdown `[issue #71](https://github.com/hartwork/image-bootstrap/issues/71)`. so there is more information that the plain link, not less. What you say regarding cut off: was that a misunderstanding? Else please elaborate.

Reply:
Being cut off: I was worried about a user having cloned the repo and not being able to resolve the full URL for research.
I guess, the real issue here is between the link text shown and the URL being embedded within the <a href>... 

Like I said: If you'd like more adjustments, go for it. My vacation's almost over and I won't be online as frequent…